statuslistener/statuslistener_test.go

```diff
@@ -248,6 +275,29 @@ func TransformDateFieldsInJSONForBulkMessage(resourceType string, resourceID str
 
 	contentMap["endpoints"] = endpoints
 
+	var applicationAuthentications []interface{}
+
+	applicationAuthenticationsBulkMessage, success := bulkMessage["application_authentications"].([]m.ApplicationAuthentication)
+
+	if success {
```

Comment:
I would suggest using what's called "the comma ok" notation, which is a convention for these cases: https://go.dev/doc/effective_go#maps . The same thing applies for the type checkings below.